### PR TITLE
Feature/leaders pages

### PR DESCRIPTION
**Pull request by MattHaslemTech**
Updates for leaders dashboard pages.

- Fix for Coach Profile reports page.
- Data from observations of teachers that were previously a part of a Coach, Site, or Program are now accounted for in the reports.
- Breadcrumbs added for the reports pages.
- Typo fixes.
- Fix issue with creating duplicate teachers.
- Validation checking for new users.
- Various design fixes.
- Fix sorting issues on Users tables

### CODE REVIEW

File: src/components/SiteProfileComponents/SiteProfile.tsx [hunk 1]
```diff
@@ -283,10 +284,12 @@ class SiteProfile extends React.Component {
     if(pageNumber == 1)
     {
```
Comment (thompchr) [inline src/components/SiteProfileComponents/SiteProfile.tsx:285]: I'm a little worried these numbers are super opaque and won't make sense later. can we at least replace these with named constants?

File: src/components/TeacherProfileComponents/TeacherProfile.tsx [hunk 2]
```diff
@@ -116,6 +116,8 @@ class TeacherProfile extends React.Component {
   componentDidMount(): void {
     // Build all initial dropdown option
     this.setDropdownOptions();
+
+    this.handlePageChange(1);
   }
```
Comment (thompchr) [inline src/components/TeacherProfileComponents/TeacherProfile.tsx:121]: Same comment

**Review by thompchr — CHANGES_REQUESTED** [review]
This looks good! Couple of comments in there about the use of magic numbers. Can we replace those at least with named constants (_i.e.,_ `const SOME_PAGE_NAME = 1`) so its more apparent to somebody reading what's happening?

File: src/components/SiteProfileComponents/SiteProfile.tsx (hunk 1 above)
Comment (MattHaslemTech) [reply src/components/SiteProfileComponents/SiteProfile.tsx:288]: Changes have been made sir!

**Review by thompchr — APPROVED** [review]
Great! nice work!

**Pull request merged**